tests/subsys/usb_c/mocks/mock_tester.c

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright 2023 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <string.h>
+#include <stddef.h>
+#include <zephyr/drivers/usb_c/usbc_pd.h>
```

Comment:
I think you might need to go over this PR with `clang-format` to sort the headers.